apple/internal/apple_framework_import.bzl

```diff
@@ -249,7 +249,7 @@ def _apple_dynamic_framework_import_impl(ctx):
         "dynamic_framework_file",
         header_imports,
         module_map_imports,
-        _all_framework_binaries(framework_groups),
+        [] if ctx.attr.avoid_link else _all_framework_binaries(framework_groups),
```

Comment:
What would we have to change in `_ios_framework_impl` to get this to work for `ios_framework` as well?

Reply:
Looks like we'll have to change this instead https://github.com/bazelbuild/rules_apple/blob/652eb8ea1802c60d7745ec16f0110ca3cf1ccddb/apple/internal/partials/framework_provider.bzl

Reply:
I pushed something to that, I haven't tested it at all.